float/src/main/java/com/novoda/floatschedule/convert/GithubUserConverter.java

```diff
@@ -0,0 +1,13 @@
+package com.novoda.floatschedule.convert;
+
+import java.util.List;
+
+public interface GithubUserConverter {
+    
+    List<String> getGithubUsers() throws Exception;
```

Comment:
does throwing such a generic exception on the interface make sense? What if someone made a really simple solution that never threw anything, the client would still have to try catch